po/da.po

```diff
@@ -42,7 +42,7 @@ msgid ""
 "Mount/unmount filesystems defined in the fstab file with the x-udisks-auth "
 "option"
 msgstr ""
-"Monter/afmonter filsystemer defineret i fstab-filen med tilvalget x-storaged-"
```

Comment:
I've checked Zanata a there are some new translations (our PO files were downloaded in October) -- I'm not sure how Zanata handles conflicts when uploading new PO files and I really don't want to risk it. I now have admin rights for storaged project in Zanata, so I can fix it manually (it's just 1 string for 6 langugages so it should be easier).